### PR TITLE
i18n(fr): Update `guides/customization.mdx` with <Steps>

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description

i18n(fr): Update `guides/customization.mdx` with <Steps> from https://github.com/withastro/starlight/commit/554c670c402c5cd87fe376d506292a8f0b017ed2

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
Merci @thomasbnt, LGTM ✅

File: docs/src/content/docs/fr/guides/customization.mdx [hunk 1]
```diff
@@ -13,13 +12,15 @@ Lorsque vous souhaitez commencer à personnaliser l’apparence de votre site St
...
+<Steps>
+
 1. Ajoutez votre fichier logo au répertoire `src/assets/` :
 
    <FileTree>
 
    - src/
      - assets/
-       - **mon-logo.svg**
+       - **my-logo.svg**
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/guides/customization.mdx:23]:
```suggestion
       - **mon-logo.svg**
```

**Review by HiDeoo — APPROVED** [review]
Thanks for the update, I'm reverting a translation from French to English for `mon-logo.svg` as it's fine in French and the `mon-logo.svg` is used in later example so we cannot revert it only in 1 place.

**Comment by thomasbnt** [comment]
Thanks @HiDeoo and @dreyfus92 💜🙌

**Pull request merged**